voxblox/src/integrator/tsdf_integrator.cc

```diff
@@ -208,6 +208,93 @@ inline float TsdfIntegratorBase::getVoxelWeight(const Point& point_C) const {
   return 0.0f;
 }
 
+void SimpleTsdfIntegrator::fillSphereAroundPoint(const Point& center,
```

Comment:
Ah - I wanted to be able to edit the TSDF so that these voxels would be below the threshold for a voxel to be considered fixed. We are abusing this functionality to mark known spaces free or occupied for offline planning, as a workaround for noisy data.
https://github.com/ethz-asl/voxblox/blob/d7d4ba485464a19891025eaea1c5e9b53f3eaa62/voxblox/src/integrator/esdf_integrator.cc#L271

This function probably makes more sense as part of Layer, then - would you agree? Then we can (ab)use it as we were doing before.

It differs from what @helenol implemented in that we want to completely clear or mark as occupied a sphere, rather than clear an inner region and mark unknown voxels in some bigger region as occupied.
https://github.com/ethz-asl/voxblox/blob/d7d4ba485464a19891025eaea1c5e9b53f3eaa62/voxblox/src/integrator/esdf_integrator.cc#L51